docs/build/web-notes.md

```diff
@@ -0,0 +1,16 @@
+# KeymanWeb configuration notes
+
+## Minimum Web Compilation Requirements
+
+* [Java 7+](https://adoptopenjdk.net/releases.html) (required by the Google Closure Compiler)
+    * This can be ignored for Keyman Developer builds.
+* A local installation of [Node.js](https://nodejs.org/) v8.9+.
+	* Builds will call `npm install` to automatically install further necessary build dependencies.
+
+	* Linux users can run the following to update to LTS version of nodejs
```

Comment:
Hmm, the current LTS of nodejs is 14 :confused:. I don't think nodejs 8 is maintained anymore.

Reply:
Pretty sure that's talking about a _minimum_ version.  Just because it's no longer maintained doesn't mean it's not compatible.